src/huggingface_hub/hf_api.py

```diff
@@ -1177,18 +1177,55 @@ def delete_repo(
 
     def update_repo_visibility(
         self,
-        name: str,
-        private: bool,
+        name: str = None,
+        private: bool = False,
         token: Optional[str] = None,
         organization: Optional[str] = None,
         repo_type: Optional[str] = None,
+        repo_id: str = None,
     ) -> Dict[str, bool]:
-        """
-        Update the visibility setting of a repository.
+        """Update the visibility setting of a repository.
+
+        Args:
+            repo_id: A user or an organization name and a repo name seperated by a
+                ``/``.
+
+                .. versionadded: 0.4.0
```

Comment:
I've never seen this before. Not sure if we want it, but if it's useful this is cool :rocket:

Reply:
IMO this is very important for third party developers who need to know which versions of the library they're compatible with. This is how `versionadded` and `versionchanged` can be rendered: https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingRegressor.html#sklearn.ensemble.HistGradientBoostingRegressor